rubrik_cdm/data_management.py

```diff
@@ -1913,9 +1913,7 @@ def get_vsphere_vm(self, name=None, is_relic=None, effective_sla_domain_id=None,
             if ((key == 'limit') or (key == 'offset')) and not isinstance(value, int):
                 raise InvalidParameterException('The limit and offset paremeter must be an integer')
 
-        query = ''
-        for key, value in parameters.items():
-            query = query + ("{}={}".format(key, value) + '&')
+        query = '&'.join(['%s=%s' % kv for kv in parameters.items()])
```

Comment:
Can we add a comment describing what this line is doing?

Reply:
Added comments for the string join

New code in: cf7e991